test/parallel/test-readline-interface.js

```diff
@@ -1082,20 +1121,22 @@ function isWarned(emitter) {
       rli.close();
     }
 
-    // Multi-line input cursor position
+    // Multi-line input cursor position and long tabs
```

Comment:
Nit: I'd prefer we add a separate test case rather than modify an existing test case.